### ISSUE TITLE
About self-supervised and pre-trained implement

### ISSUE THREAD

**Issue by hanktseng131415go** [talreiss/PANDA#2]
Hi,

Thank you for sharing the implemented code with the paper. 

As I try to get familiar with this field, I have a few questions about training and testing on the one-class Cifar-10 dataset. From Table 2 in sec 4.3, you mentioned that self-supervised methods didn't outperform the pre-trained ones, so you believe that using pre-trained methods is one of the cores of your work. In this case, is it legit that training a model on regular Cifar-10, which is standard Cifar-10 image classification, and then testing it on one-class Cifar-10 (maybe the criterion and Average ROC AUC are following from your code)? 

Also, following the process, is it self-supervised or pre-trained by your knowledge? 

Finally, correcting if I am wrong, I notice that the implemented code only evaluate one specific class at a time. Therefore, if I want to get the same Average ROC AUC% as Table 2, I have to average the ROC AUC from every class, right? For example, class 1: 90%, class 2: 91%, class 3: 90% class 4: 90% so the average ROC AUC is 90.5% by (90% + 91% + 90% + 91%)/4.

Many Thanks

**Comment by NivC** [comment]
Hi,

Thank you for your interest in our work!

" is it legit that training a model on regular Cifar-10, which is standard Cifar-10 image classification, and then testing it on one-class Cifar-10"
Not really. In our setting we do not assume labels on the anomaly detection dataset. If you have Cifar10 labels, you may simply uses standard classification and not anomaly detection.

"Also, following the process, is it self-supervised or pre-trained by your knowledge?"
Not sure I understand the question. We start with pretrained features and adapt them in a self supervised way.

"I have to average the ROC AUC from every class, right? "
Right.

Niv.
